lib/presentation/utils/misc.dart

```diff
@@ -31,9 +34,69 @@ DataResponse getAttributeById(String id, String type, List<DataResponse> listOfD
 
 LinkResponse combineLinks(LinkResponse baseLinks, LinkResponse selfLinks) {
   return LinkResponse((b) => b
-      ..first = baseLinks?.first
-      ..last = baseLinks?.last
-      ..next = baseLinks?.next
-      ..self = selfLinks?.self
+    ..first = baseLinks?.first
+    ..last = baseLinks?.last
+    ..next = baseLinks?.next
+    ..self = selfLinks?.self);
+}
+
+List<Contact> getContacts() {
+  return List<Contact>.generate(6, (int index)=> Contact());
```

Comment:
I added a constructor and tried the following contact:

`Contact("Teste", "https://upload.wikimedia.org/wikipedia/commons/thumb/6/66/An_up-close_picture_of_a_curious_male_domestic_shorthair_tabby_cat.jpg/1920px-An_up-close_picture_of_a_curious_male_domestic_shorthair_tabby_cat.jpg", "Info", ContactType.EMAIL)`

And the following error was thrown;

```
I/flutter (26018): ══╡ EXCEPTION CAUGHT BY IMAGE RESOURCE SERVICE ╞════════════════════════════════════════════════════
I/flutter (26018): The following ArgumentError was thrown resolving an image codec:
I/flutter (26018): Invalid argument(s): No host specified in URI file:///
I/flutter (26018): 
I/flutter (26018): When the exception was thrown, this was the stack:
I/flutter (26018): #0      _HttpClient._openUrl (dart:_http/http_impl.dart:2187:9)
I/flutter (26018): #1      _HttpClient.getUrl (dart:_http/http_impl.dart:2118:48)
I/flutter (26018): #2      NetworkImage._loadAsync (package:flutter/src/painting/_network_image_io.dart:84:59)
I/flutter (26018): <asynchronous suspension>
I/flutter (26018): #3      NetworkImage.load (package:flutter/src/painting/_network_image_io.dart:48:14)
I/flutter (26018): #4      ImageProvider.resolve.<anonymous closure>.<anonymous closure>.<anonymous closure> (package:flutter/src/painting/image_provider.dart:316:48)
I/flutter (26018): #5      ImageCache.putIfAbsent (package:flutter/src/painting/image_cache.dart:160:22)
I/flutter (26018): #6      ImageProvider.resolve.<anonymous closure>.<anonymous closure> (package:flutter/src/painting/image_provider.dart:316:25)
I/flutter (26018): #7      SynchronousFuture.then (package:flutter/src/foundation/synchronous_future.dart:38:29)
I/flutter (26018): #8      ImageProvider.resolve.<anonymous closure> (package:flutter/src/painting/image_provider.dart:313:11)
I/flutter (26018): #12     ImageProvider.resolve (package:flutter/src/painting/image_provider.dart:305:16)
I/flutter (26018): #13     _ImageState._resolveImage (package:flutter/src/widgets/image.dart:951:20)
I/flutter (26018): #14     _ImageState.didChangeDependencies (package:flutter/src/widgets/image.dart:908:5)
I/flutter (26018): #15     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4086:12)
I/flutter (26018): #16     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #17     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #18     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #19     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #20     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #21     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #22     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #23     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #24     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #25     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #26     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #27     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #28     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #31     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #32     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #33     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #34     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #35     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #36     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #37     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #39     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #40     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #41     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #42     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #43     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #44     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #45     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #46     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #47     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #48     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #49     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #50     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #51     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #52     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #53     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #54     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #55     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (26018): #56     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #57     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #58     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #59     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #60     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #61     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #62     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #63     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #64     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #65     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #66     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #67     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #68     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #69     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #70     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #71     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (26018): #72     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #73     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #74     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #75     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #76     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #77     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #78     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #79     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #80     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #81     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #82     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #83     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #84     ParentDataElement.mount (package:flutter/src/widgets/framework.dart:4314:11)
I/flutter (26018): #85     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #86     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #87     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #88     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #89     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #90     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (26018): #91     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #92     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #93     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #94     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (26018): #95     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (26018): #96     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (26018): #97     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (26018): #98     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (26018): #99     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (26018): #100    SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:1181:36)
I/flutter (26018): #101    SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:1166:20)
I/flutter (26018): #102    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2328:19)
I/flutter (26018): #103    SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:1159:11)
I/flutter (26018): #104    RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:354:23)
I/flutter (26018): #105    RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:1797:58)
I/flutter (26018): #106    PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:875:15)
I/flutter (26018): #107    RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:1797:13)
I/flutter (26018): #108    RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:343:5)
I/flutter (26018): #109    RenderSliverMultiBoxAdaptor.addInitialChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:427:5)
I/flutter (26018): #110    RenderSliverGrid.performLayout (package:flutter/src/rendering/sliver_grid.dart:543:12)
I/flutter (26018): #111    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #112    RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:181:11)
I/flutter (26018): #113    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #114    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:406:13)
I/flutter (26018): #115    RenderShrinkWrappingViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1681:12)
I/flutter (26018): #116    RenderShrinkWrappingViewport.performLayout (package:flutter/src/rendering/viewport.dart:1644:20)
I/flutter (26018): #117    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #118    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #119    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #120    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #121    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #122    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #123    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #124    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #125    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #126    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #127    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #128    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #129    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #130    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #131    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #132    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #133    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #134    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #135    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #136    RenderPadding.performLayout (package:flutter/src/rendering/shifted_box.dart:206:11)
I/flutter (26018): #137    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #138    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #139    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #140    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #141    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #142    RenderSliverMultiBoxAdaptor.insertAndLayoutChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:492:13)
I/flutter (26018): #143    RenderSliverList.performLayout.advance (package:flutter/src/rendering/sliver_list.dart:195:19)
I/flutter (26018): #144    RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:238:19)
I/flutter (26018): #145    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #146    RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:181:11)
I/flutter (26018): #147    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #148    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:406:13)
I/flutter (26018): #149    RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1352:12)
I/flutter (26018): #150    RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:1270:20)
I/flutter (26018): #151    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #152    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #153    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #154    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #155    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #156    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #157    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #158    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #159    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #160    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #161    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #162    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #163    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #164    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #165    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #166    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #167    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #168    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #169    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #170    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:142:11)
I/flutter (26018): #171    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:444:7)
I/flutter (26018): #172    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:212:7)
I/flutter (26018): #173    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:356:14)
I/flutter (26018): #174    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #175    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #176    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #177    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #178    _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1232:11)
I/flutter (26018): #179    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #180    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #181    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #182    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #183    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #184    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #185    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #186    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #187    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #188    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #189    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #190    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #191    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #192    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (26018): #193    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #194    RenderOffstage.performLayout (package:flutter/src/rendering/proxy_box.dart:3138:14)
I/flutter (26018): #195    RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
I/flutter (26018): #196    RenderStack.performLayout (package:flutter/src/rendering/stack.dart:510:15)
I/flutter (26018): #197    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1578:7)
I/flutter (26018): #198    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:844:18)
I/flutter (26018): #199    RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:341:19)
I/flutter (26018): #200    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:761:13)
I/flutter (26018): #201    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:280:5)
I/flutter (26018): #202    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1033:15)
I/flutter (26018): #203    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:975:9)
I/flutter (26018): #204    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:891:5)
I/flutter (26018): #208    _invoke (dart:ui/hooks.dart:249:10)
I/flutter (26018): #209    _drawFrame (dart:ui/hooks.dart:207:3)
I/flutter (26018): (elided 6 frames from package dart:async)
I/flutter (26018): 
I/flutter (26018): Image provider: NetworkImage("", scale: 1.0)
I/flutter (26018): Image key: NetworkImage("", scale: 1.0)
I/flutter (26018): ════════════════════════════════════════════════════════════════════════════════════════════════════
```

Reply:
The issue is resolved. The image will load from the URL specified in the constructor. 

![vost-pt-kitty](https://user-images.githubusercontent.com/34758667/67123039-d050db00-f20c-11e9-8626-86ecbc8dc201.jpeg)